addons/udes_stock/models/stock_move_line.py

```diff
@@ -1037,3 +1037,23 @@ def _prevent_write_after_done(self):
         done_lines = self.filtered(lambda x: x.state == "done")
         if done_lines:
             raise ValidationError(_("Cannot update move lines that are already 'done'."))
+
+    @api.constrains("result_package_id", "u_result_parent_package_id")
+    def _check_resultant_package_level(self):
+        pack_level = self.result_package_id.u_package_depth
+
+        # Assumes that the current package hierarchy is acceptable
+        if pack_level > max(self.u_result_parent_package_id.children_ids.mapped("u_package_depth")):
```

Comment:
can't we just use the u_package_depth on the u_result_parent_package_id? Why do the max? This is already the max isn't it? Or does the package not exist yet/this is before the children haven't been put in?